### PR TITLE
KOGITO-5637 Add LTS check to BDD nightly 

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -331,45 +314,44 @@ pipeline {
...
+    return JOBS[jobKey]
 }
 
-String getJobUrl(String jobName) {
-    echo "getJobUrl for ${jobName}"
-    def job = getJob(jobName)
+String getJobUrl(String jobKey) {
+    echo "getJobUrl for ${jobKey}"
+    def job = getJob(jobKey)
     return job ? job.getAbsoluteUrl() : ''
```
Comment (Ginxo) [inline Jenkinsfile.nightly]:
```suggestion
    return getJob(jobKey)?.getAbsoluteUrl() ?: ''
```

File: Jenkinsfile.nightly [hunk 2]
```diff
@@ -331,45 +314,44 @@ pipeline {
...
+def getJob(String jobKey) {
+    return JOBS[jobKey]
 }
 
-String getJobUrl(String jobName) {
-    echo "getJobUrl for ${jobName}"
-    def job = getJob(jobName)
+String getJobUrl(String jobKey) {
+    echo "getJobUrl for ${jobKey}"
+    def job = getJob(jobKey)
```
Comment (Ginxo) [inline Jenkinsfile.nightly]:
```suggestion
```

File: Jenkinsfile.nightly [hunk 3]
```diff
@@ -331,45 +314,44 @@ pipeline {
...
     return job ? job.getAbsoluteUrl() : ''
 }
 
-boolean isJobSucceeded(String jobName) {
-    def job = getJob(jobName)
+boolean isJobSucceeded(String jobKey) {
+    def job = getJob(jobKey)
```
Comment (Ginxo) [inline Jenkinsfile.nightly]:
```suggestion
```

File: Jenkinsfile.nightly [hunk 4]
```diff
@@ -331,45 +314,44 @@ pipeline {
...
     return job ? job.getResult() == 'SUCCESS' : false
```
Comment (Ginxo) [inline Jenkinsfile.nightly]:
```suggestion
    return getJob(jobKey)?.getResult() == 'SUCCESS'
```

File: Jenkinsfile.nightly [hunk 5]
```diff
@@ -405,10 +387,10 @@ void sendPipelineErrorNotification() {
                 to: env.KOGITO_CI_EMAIL_TO
 }
 
-List getDefaultBuildParams(String buildBranchName = '') {
+List getDefaultBuildParams(String buildBranchName = '', String key = '') {
     buildBranchName = buildBranchName ?: getBuildBranch()
     List params = []
-    addStringParam(params, 'DISPLAY_NAME', env.NIGHTLY_TAG)
+    addStringParam(params, 'DISPLAY_NAME', "${key ? key + '-' : ''}${env.NIGHTLY_TAG}")
```
Comment (Ginxo) [inline Jenkinsfile.nightly]:
```suggestion
    addStringParam(params, 'DISPLAY_NAME', "${key ? "${key}-" : ''}${env.NIGHTLY_TAG}")
```

You can try 
```
def job = 'jobvalue'
def key = 'key1'

println "${job ? "${key}-" : 'aaa'}"
```
at https://groovyconsole.appspot.com/ to check string replacement is properly working

**Review by Ginxo — APPROVED** [review]
LGTM, just few technical suggestions

File: Jenkinsfile.nightly (hunk 5 above)
Comment (radtriste) [reply Jenkinsfile.nightly]: I wasn't sure about that. Thanks for the link !!! :)

**Comment by radtriste** [comment]
Depends on https://github.com/kiegroup/kogito-operator/pull/976 before merging

**Pull request merged**